XmlSchemaClassGenerator.Console/Program.cs

```diff
@@ -87,6 +88,7 @@ static void Main(string[] args)
                 { "r|order", "emit order for all class members stored as XML element", v => emitOrder = v != null },
                 { "c|pcl", "PCL compatible output", v => pclCompatible = v != null },
                 { "p|prefix=", "the {PREFIX} to prepend to auto-generated namespace names", v => namespacePrefix = v },
+                { "po|prefixOnOutput", "include the prefix on the path (default is enabled)", v => addPrefixToOutputPath = v != null },
```

Comment:
IMO the phrasing can be improved here. Perhaps: `"po|prefixPath", "include namespace prefix in output paths (default is enabled)"`